app/bundles/LeadBundle/Tests/EventListener/DashboardSubscriberTest.php

```diff
@@ -0,0 +1,409 @@
+<?php
+
+namespace Mautic\LeadBundle\Tests\EventListener;
+
+use Mautic\DashboardBundle\Event\WidgetDetailEvent;
+use Mautic\LeadBundle\EventListener\DashboardSubscriber;
+use Mautic\LeadBundle\Model\LeadModel;
+use Mautic\LeadBundle\Model\ListModel;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Routing\RouterInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class DateHelperStub
```

Comment:
We should have one class per file. Please move this to a separate file.

Reply:
Should `DateHelperStub.php` be created under `Tests/Services` ?

I also see that `app/bundles/LeadBundle/Tests/Services/PeakInteractionTimerTest.php` contains two classes. I'm not familiar with the standards, but is it acceptable to have two classes in that file?